bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,21 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	if [[ ! -d "${ED}"/etc/init.d ]] ; then
+		continue
+	fi
+
+	for x in "${ED}"/etc/init.d/* ; do
+		if [[ -f "${ED}"/etc/init.d/* ]] ; then
+			:
```

Comment:
This is still a meaningless statement. I think you really want to use nullglob there.